convert2rhel/unit_tests/checks_test.py

```diff
@@ -813,3 +825,86 @@ def test_custom_repos_are_invalid(self):
         self.assertEqual(len(checks.logger.critical_msgs), 1)
         self.assertEqual(len(checks.logger.info_msgs), 0)
         self.assertTrue("Unable to access the repositories passed through " in checks.logger.critical_msgs[0])
+
+
+@pytest.mark.parametrize(
+    ("package", "return_code", "raise_system_exit"),
+    (
+        (
+            "convert2rhel.noarch   0.24-1.20211019110627581338.pr343.11.g4ff8753.el8   copr:copr.fedorainfracloud.org:group_oamg:convert2rhel\n"
+            "convert2rhel.src      0.24-1.20211019110627581338.pr343.11.g4ff8753.el8   copr:copr.fedorainfracloud.org:group_oamg:convert2rhel\n"
+            "tzdata.noarch         2021c-1.el8                                         baseos\n",
+            0,
+            True,
+        ),
+        ("", 0, False),
+    ),
+)
+def test_check_package_updates(package, return_code, raise_system_exit, monkeypatch, caplog):
+    run_subprocess_mocked = mock.Mock(
+        spec=run_subprocess,
+        side_effect=run_subprocess_side_effect(
+            (
+                (
+                    "yum",
+                    "check-update",
+                    "-q",
+                ),
+                (package, return_code),
+            ),
+        ),
+    )
+    monkeypatch.setattr(
+        checks,
+        "run_subprocess",
+        value=run_subprocess_mocked,
+    )
+
+    if raise_system_exit:
+        with pytest.raises(SystemExit):
+            assert check_package_updates()
```

Comment:
After the raises, it would be good to check if the amount of packages is accurate. `The system has %s packages to be updated`

Reply:
Updated that assert as well.